### ISSUE TITLE
Temperature control

### ISSUE THREAD

**Issue by shamangary** [zou-group/textgrad#104]
Hi first of all super cool library, I find it very useful and inspiring.

I am playing with it and found there is no control sometimes in terms of temperature.

Such as the backward engine, and the blackbox LLM.

Do you mind to let me know how to do this or maybe you have plan to improve it?

Thank you. Cool work.

**Comment by mertyg** [comment]
Good point! It should be relatively straightforward to add this, e.g., adding an argument to the forward call and passing [here](https://github.com/zou-group/textgrad/blob/main/textgrad/autograd/llm_ops.py#L60).
